docs/commcare-cloud/deploy-branches.md

```diff
@@ -0,0 +1,100 @@
+# Deploying specific branches
+
+In some cases it may be desirable to have full control over what code is deployed. This can
+be accomplished by creating a `deploy_branches.yml` configuration file for you environment.
+This file defined exactly which branches will get deployed.
+
+The format of the file is as follows:
+
+```yaml
+trunk: master
+name: autostaging
+branches:
+  - feature1
+  - feature2
+  - forkowner:feature3 # branch from fork of repository
+submodules:
+  submodules/module1:
+    branches:
+      - feature1
+      - forkowner:feature2 # branch from fork of repository
+  submodules/module2:
+    trunk: develop
+    branches:
+      - feature2
+```
+When not specified, a submodule's trunk and name inherit from the parent.
+
+Multiple repositories can also be specified:
+
+```yaml
+   dimagi/commcare-hq:  # this repo is required
+     trunk: master
+     ...
+
+   another/repo:
```

Comment:
Know if it'd be useful for ICDS to be able to rebuild only one of these repos?  For example, I could see it being useful for someone without write access to the HQ repo to still be able to rebuild the ICDS repo in isolation.

Reply:
it may be useful in future but we can cross that bridge when we get there